tests/Lykke.Service.QuotesHistory.Tests/QuoteHistoryRepositoryTests.cs

```diff
@@ -19,7 +19,9 @@ public class QuoteHistoryRepositoryTests
         [Fact(Skip = "Hangs in TC")]
         public void RepositoryCanStoreQuotesWithSameTimestamp()
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
1. Use EmptyLogFactory.Instance in such cases.
2. Remove Assert.Equal(0, log.Count). It is a bad practice to use log for asserts.
3. CreateStorage implementation not uses log at all, so assertion is useless anyway.